portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro

```diff
@@ -125,6 +125,29 @@ definition {
 			value1 = "${key_untracked}");
 	}
 
+	macro checkWorkflowReports {
+
+		ProductMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Control Panel",
+			portlet = "Metrics");
+
+		AssertTextEquals(
+			key_workflowProcessName = "Single Approver",
+			locator1 = "WorkflowMetrics#METRICS_WORKFLOW_PROCESS_VALUE_OVERDUE",
+			value1 = "${key_overdueValue}");
+		AssertTextEquals(
+			key_columnName = "ON_TIME",
+			key_workflowProcessName = "Single Approver",
+			locator1 = "WorkflowMetrics#METRICS_WORKFLOW_PROCESS_VALUE_ON_TIME",
+			value1 = "${key_onTimeValue}");
+		AssertTextEquals(
+			key_columnName = "TOTAL_PENDING",
+			key_workflowProcessName = "Single Approver",
+			locator1 = "WorkflowMetrics#METRICS_WORKFLOW_PROCESS_VALUE_TOTAL_PENDING",
+			value1 = "${key_totalPendingValue}");
```

Comment:
Do you think that Untracked should be added here as well?

Reply:
I can add it. Should be an easy addition!

Reply:
Thank you!